test/library/draft/arrow/Arrow.chpl

```diff
@@ -0,0 +1,253 @@
+module Arrow {
+    require "ArrowDecl.chpl";
+    public use ArrowDecl;
+    public use SysCTypes, CPtr;
+    // -------------------------- Type Declarations and Functions -------------
+
+    type ArrowArray = GArrowArray;
+
+    proc arrowInt32(): GArrowInt32DataType {
+        return garrow_int32_data_type_new();
+    }
+    
+    proc arrowInt64(): GArrowInt32DataType {
+        return garrow_int64_data_type_new();
+    }
+    proc arrowString(): GArrowStringDataType {
+        return garrow_string_data_type_new();
+    }
+    proc arrowBool(): GArrowBooleanDataType {
+        return garrow_boolean_data_type_new();
+    }
+    //--------------------------- Array building functions --------------------
+
+    proc array(array: [?arrDom] ?arrayType, validity: [?validityDom] int = [-1], 
+                specifiedType = nil): GArrowArray {
+        var fakeReturn: GArrowArray;
```

Comment:
I would think there would be a better way to do this (where you don't need to actually create the variable and can just have it return something of that type), but I am not sure off the top of my head. If we do decide to go this way though, I would move this line to be in the `otherwise` block, so it's not just a random variable created that isn't used most of the time.

Reply:
Right now I am just putting it inside the otherwise block. But I think we should probably throw an error here once we discuss error handling.